### PR TITLE
RN RHEL9 openshift-install support

### PR DESCRIPTION
**Pull request by bscott-rh**
RN for adding RHEL9 openshift-install support.
4.11 only
Doc preview: https://bscott-rh.github.io/openshift-docs/RN-RHEL9-openshift-installer/release_notes/ocp-4-11-release-notes.html#ocp-4-11-rhel-9-openshift-installer

### CODE REVIEW

**Comment by bscott-rh** [comment]
@gpei  PTAL at this release note for RHEL9 and openshift installer. Thank you

**Comment by gpei** [comment]
LGTM, thanks

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -1337,8 +1343,7 @@ With this update, the `user-workload-monitoring-config` config map is now automa
...
+* Before this update, for user workload monitoring, if you configured external labels for metrics in Prometheus, the CMO did not correctly propagate these labels to Thanos Ruler. If you queried external metrics for user-defined projects, not provided by the user workload monitoring instance of Prometheus, you would sometimes not see external labels for these metrics even though you had configured Prometheus to add them. With this update, the CMO now properly propagates the external labels that you configured in Prometheus to Thanos Ruler, and you can see the labels when you query external metrics. Thus, for user-defined projects, if you queried external metrics not provided by the user workload monitoring instance of Prometheus, you would sometimes not see external labels for these metrics even though you had configured Prometheus to add them. With this update, the CMO now properly propagates the external labels that you configured in Prometheus to Thanos Ruler, and you can see the labels when you query external metrics.
```
Comment (tmalove) [inline release_notes/ocp-4-11-release-notes.adoc:1346]:
```suggestion
* Before this update, for user workload monitoring, if you configured external labels for metrics in Prometheus, the CMO did not correctly propagate these labels to Thanos Ruler. If you queried external metrics for user-defined projects, not provided by the user workload monitoring instance of Prometheus, you would sometimes not see external labels for these metrics even though you had configured Prometheus to add them. With this update, the CMO now properly propagates the external labels that you configured in Prometheus to Thanos Ruler, and you can see the labels when you query external metrics. Therefore, for user-defined projects, if you queried external metrics not provided by the user workload monitoring instance of Prometheus, you would sometimes not see external labels for these metrics even though you had configured Prometheus to add them. With this update, the CMO now properly propagates the external labels that you configured in Prometheus to Thanos Ruler, and you can see the labels when you query external metrics.
```

Comment (tmalove) [reply release_notes/ocp-4-11-release-notes.adoc:1346]: Ben, for some reason my update was not highlighted...I changed 'thus' to 'therefore' in the paragraph within line 1346. I cannot find the style guide source for this (yet), but it is in the 'terms error' list.

**Pull request merged**